File: templates/JavaScript/template.js
const fs = require('fs');

let ans1 = 0;
let ans2 = 0;
const lines = fs.readFileSync(process.argv[2], 'utf-8').trim().split('\n');
lines.forEach(line => {
  console.log(line);
});

console.log("Part 1: " + ans1);
console.log("Part 2: " + ans2);

File: templates/Python/template.py
import re
import sys
import math
from copy import deepcopy
from collections import deque

ans1, ans2 = 0, 0
lines = open(sys.argv[1]).read().split('\n')
for line in lines:
    print(line)

print(f'Answer Part 1: {ans1}')
print(f'Answer Part 2: {ans2}')